lib/lib_rss.php

```diff
@@ -823,25 +823,17 @@ function errorMessageInfo($errorTitle, $error = '') {
 		$details = "<pre>{$details}</pre>";
 	}
 
+	header('HTTP 1.1 500 Internal Server Error', true, 500);
+	header("Content-Security-Policy: default-src 'self'");
```

Comment:
Similar to comment just above, except that I am not sure of where to best put that line, so maybe just leave it there for now